.github/workflows/pre_merge.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Code quality checks
         run: tox -vv -e pre-commit
   Unit-Test:
-    runs-on: [self-hosted, linux, x64, dev]
+    runs-on: ubuntu-20.04
```

Comment:
Do we need to specify the ubuntu version ? If so, I prefer to 22.04

Reply:
It's just for choosing the Linux OS from the GH provide actions-runner. I think this is the first step to use GH-hosted runner for the unttests pipeline so that could be changed or added other version of OSes later.